Content.Server/Botany/Systems/SeedExtractorSystem.cs

```
@@ -27,7 +27,12 @@ private void OnInteractUsing(EntityUid uid, SeedExtractorComponent seedExtractor
 
         if (!TryComp(args.Used, out ProduceComponent? produce))
             return;
-        if (!_botanySystem.TryGetSeed(produce, out var seed) || seed.Seedless)
+
+        if (!_botanySystem.TryGetSeed(produce, out var seed))
```

Why change this to fail silently?